charts/mageai/templates/tests/test-connection.yaml

```diff
@@ -11,5 +11,5 @@ spec:
     - name: wget
       image: busybox
       command: ['wget']
-      args: ['{{ include "mageai.fullname" . }}:{{ .Values.service.port }}']
+      args: ['{{ include "mageai.fullname" . }}:{{ .Values.service.port }}{{ include "mageai.basePath" . }}']
```

Comment:
I think you need to add a `/` between port and basePath

Reply:
Hi @wangxiaoyou1993  I am adding it as part of the mageai.basePath definition on _helpers.tpl

Reply:
got it.